ImageProcessing/Re-RunHSC.ipynb

```diff
@@ -0,0 +1,271 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# HSC Re-Run: Making Forced Photometry Light Curves from Scratch\n",
+    "\n",
+    "<br>Owner: **Justin Myles** ([@jtmyles](https://github.com/LSSTScienceCollaborations/StackClub/issues/new?body=@jtmyles))\n",
+    "<br>Last Verified to Run: **N/A -- in development**\n",
+    "\n",
+    "This project addresses issue [#63: HSC Re-run](https://github.com/LSSTScienceCollaborations/StackClub/issues/63)\n",
+    "\n",
+    "This notebook demonstrates the [LSST Science Piplines data processing tutorial](https://pipelines.lsst.io/) with emphasis on how a given [obs package](https://github.com/lsst/obs_base) works under the hood of the command line tasks. It makes use of the [obs_subaru](https://github.com/lsst/obs_subaru) package to measure a forced photometry light curve for a small patch in the HSC sky in the [ci_hsc](https://github.com/lsst/ci_hsc/) repository. \n",
+    "\n",
+    "### Learning Objectives:\n",
+    "After working through and studying this notebook you should be able to understand how to use the DRP pipeline from image visualization through to a forced photometry light curve. Specific learning objectives include: \n",
+    "   1. [configure command-line tasks](https://pipelines.lsst.io/v/w-2018-12/modules/lsst.pipe.base/command-line-task-config-howto.html) for your science case\n",
+    "   2. TODO\n",
+    "   \n",
+    "Other techniques that are demonstrated, but not empasized, in this notebook are\n",
+    "   1. Use the `butler` to fetch data\n",
+    "   2. Visualize data with the LSST Stack\n",
+    "   3. TODO\n",
+    "\n",
+    "### Logistics\n",
+    "This notebook is intended to be runnable on `lsst-lspdev.ncsa.illinois.edu` from a local git clone of https://github.com/LSSTScienceCollaborations/StackClub.\n",
+    "\n",
+    "\n",
+    "## Set Up"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "import eups.setupcmd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Part I: Interacting with data. Introduction to the Butler\n",
+    "https://pipelines.lsst.io/getting-started/data-setup.html\n",
+    "\n",
+    "Part I runs the following command-line tasks\n",
```

Comment:
I think you need to explain what you are going to do here (or above). Are you going to execute the command line tasks, using `!` or `%%bash` in the cells below? Or are you going to unpack each command line task into its python components and run them as python commands, as if you were writing a single python script to do everything?